public/_redirects

```diff
@@ -0,0 +1,2 @@
+http://liquidity.dfi.dev/* https://uniswap.defichain.com/:splat 301!
```

Comment:
Following this - https://github.com/DeFiCh/scan/blob/ff8013338d9300d75799b7ef3da0b840c6540c78/public/_redirects